client/core/core.go

```diff
@@ -3152,30 +3231,9 @@ func (c *Core) initialize() {
 		wg.Add(1)
 		go func(acct *db.AccountInfo) {
 			defer wg.Done()
-			host, err := addrHost(acct.Host)
-			if err != nil {
-				c.log.Errorf("skipping loading of %s due to address parse error: %v", acct.Host, err)
-				return
-			}
-			dc, err := c.connectDEX(acct)
-			if err != nil {
-				c.log.Errorf("error connecting to DEX %s: %v", acct.Host, err)
+			if !c.verifyAccount(acct) {
```

Comment:
I like the refactoring here, but it does raid the question of what we should do upon importing an account with active orders.  Before authorizing with the imported dex, we'd want to run all the other code in initialize like loading wallets and db trades to that when authorizing with dex, the orders are all loaded up.

Reply:
Also good for a part 2